src/main.jsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
```

Comment:
Why is this not TypeScript?

Reply:
Because it was easier to not have TypeScript for the first trial. I was thinking of adding it after the first working trial

Reply:
OK no worries. We'll make it `tsx` before merging then.

Reply:
Done (seems that just the types were missing)